storage/src/main/java/com/tencent/rss/storage/common/LocalStorage.java

```diff
@@ -87,6 +88,11 @@ public String getStoragePath() {
     return basePath;
   }
 
+  @Override
+  public String getStorageHost() {
+    return STORAGE_HOST;
```

Comment:
maybe server's ip will be better.

Reply:
According to definition of URI, host is part of it, but ip doesn't.

Reply:
server's hostname?

Reply:
there has logical when update metrics:
```
if (hostName = 'localhost') {
   updateConstantMetrics
} else {
   updateDynamicMetrics
}
```
That's why make storage host as constant for LocalStorage.

Reply:
OK